app/lib/common/widgets/scrollable_list_tab_scroller.dart

```diff
@@ -363,37 +365,51 @@ class DefaultHeaderWidget extends StatefulWidget {
 }
 
 class _DefaultHeaderWidgetState extends State<DefaultHeaderWidget>
-    with SingleTickerProviderStateMixin {
-  late final TabController _tabController;
+    with TickerProviderStateMixin {
+  TabController? _tabController;
 
   @override
   void initState() {
     super.initState();
+    _setController();
+  }
+
+  void _setController() {
+    _tabController?.dispose();
+    widget.selectedTabIndex.removeListener(externalTabChangeListener);
```

Comment:
I think this is unnecessary as these resources are intended to be in ```dispose()``` method. Can you confirm it's needed?

Reply:
no, that is incorrect. see above.